cmd/ocm/login/cmd.go

```diff
@@ -143,6 +146,13 @@ func init() {
 			"this option is provided then the user name and password will be stored "+
 			"persistently, in clear text, which is potentially unsafe.",
 	)
+	flags.BoolVar(
+		&args.useAuthCode,
+		"use-auth-code",
+		false,
+		"Enables login using OAuth2, Redirects user to Red Hat SSO.",
```

Comment:
We should update the help text here, even if it will be hidden to start. Just want to make sure we mention Authorization Code login.

Perhaps we should also mention about logging out, i.e. navigating to sso.redhat.com and logging out manually in order to change accounts like we had to demo in the rosa office hours

Reply:
Changed the message